docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -150,7 +150,83 @@ You should see the URL for your bootstrap package as the value for `mdm.macos_se
 
 ## macOS Setup Assistant
 
-> This feature is currently in development.
+When an end user unboxes their new Mac, or starts up a freshly wiped Mac, they're presented with the macOS Setup Assistant. Here they see panes that allow them to configure accessibility, appearance, and more.
+
+In Fleet, you can customize the macOS Setup Assistant by using an automatic enrollment profile.
+
+To customize the macOS Setup Assistant, we will do the following steps:
+
+1. Create an automatic enrollment profile
+2. Upload the profile to Fleet
+3. Test the custom macOS Setup Assistant
+
+### Step 1: create an automatic enrollment profile
+
+1. Download Fleet's example automatic enrollment profile by navigating to the example [here on GitHub](https://github.com/fleetdm/fleet/blob/main/mdm_profiles/setup_assistant.json) and clicking the download icon.
+
+2. Open the automatic enrollment profile and replace the `profile_name` key with your organization's name.
+
+3. View the the list of macOS Setup Assistant properties (panes) [here in Apple's Device Management documentation](https://developer.apple.com/documentation/devicemanagement/skipkeys) and choose which panes to hide from your end users.
+
+4. In your automatic enrollment profile, edit the `skip_setup_items` array so that it includes the panes you want to hide.
+
+> You can modify properties other than `skip_setup_items`. These are documented by Apple [here](https://developer.apple.com/documentation/devicemanagement/profile).
+
+### Step 2: upload the profile to Fleet
```

Comment:
Built the docs locally to preview this since there's a lot of new content, and ran into some formatting issues with this section.

As-is, the step numbers are indented but everything beneath is not:
<img width="901" alt="Screenshot 2023-07-17 at 7 44 41 PM" src="https://github.com/fleetdm/fleet/assets/3065949/21aaff82-89d0-4231-a07d-b89e5be3f8ee">


When you add spaces to align the content, it almost works, but it looks like our markdown parser can't handle a 2nd code block in an ordered list:
<img width="856" alt="Screenshot 2023-07-17 at 7 43 40 PM" src="https://github.com/fleetdm/fleet/assets/3065949/aa59ba65-0c20-452f-9314-3e2223fd7567">


@noahtalerman what do you think? Could the 2 code blocks maybe be combined into 1 with a comment explaining each section? or maybe could use numbered headings instead of an ordered list?

Reply:
Hmm, good catch. I prefer not to combine the code blocks because we want the user to be able to quickly copy/paste the blocks separately.

Numbered headings is another good idea however we're using ordered lists and run into the same misalignment issue elsewhere in the docs. For example, the custom macOS settings page [here](https://fleetdm.com/docs/using-fleet/mdm-custom-macos-settings#step-2-upload-configuration-profile-to-fleet).

Given the above, I think these are ok to merge with the misalignment. What do you think?

> it looks like our markdown parser can't handle a 2nd code block in an ordered list

How difficult is it to update the markdown parser? cc @eashaw 

We could follow up with a website request for this.